src/main/webapp/WEB-INF/tags/page.tag

```diff
@@ -19,10 +19,6 @@
     <link type="text/css" href="<%= FrontEndLibrary.BOOTSTRAP_THEME_CSS %>" rel="stylesheet">
     <link type="text/css" href="/stylesheets/teammatesCommon.css" rel="stylesheet">
     <jsp:invoke fragment="cssIncludes" />
-    <!--[if lt IE 9]>
-      <script src="https://cdn.jsdelivr.net/html5shiv/3.7.3/html5shiv.min.js"></script>
-      <script src="https://cdn.jsdelivr.net/respond/1.4.2/respond.min.js"></script>
-    <![endif]-->
```

Comment:
Just curious: did TEAMMATES use to work on IE8 before this?

Reply:
Honestly, I can't say. I have never explored the area of browser compatibility here.